_meta/config/common.reference.p2.yml.tmpl

```diff
@@ -60,6 +60,9 @@ inputs:
 #     #reporting_threshold: 10000
 #     # Frequency used to check the queue of events to be sent out to fleet.
 #     #reporting_check_frequency_sec: 30
+    server:
+      # Sets the the to wait for setup when elastic-agent first starts an instance of fleet-server
+      fleet_timeout: 30s
```

Comment:
`fleet.server.*` settings cannot come from the configuration file, they must come from the command line for `install` and `enroll`. This is because the elastic-agent checks for the `fleet.server.*` settings to determine to run a Fleet Server.

Reply:
Yah, wasn't entirely sure I was doing that right, but that makes sense.